### PR TITLE
Coverage test utils

### LINKED ISSUE #392: Coverage: increase coverage for tx_utils
https://app.codecov.io/gh/Badger-Finance/badger-rewards/blob/development/rewards/utils/tx_utils.py

### PR DESCRIPTION
**Pull request by SHAKOTN**
**Done:**
1. Add full coverage for `test_utils.py` module
2. Replace `http` with `http_client` in `get_effective_gas_price`
3. Package tests to separate python packages for easy navigation

Fixes #392

### CODE REVIEW

File: tests/test_utils/test_tx_utils.py [hunk 1]
```diff
@@ -111,3 +113,49 @@ def test_get_latest_base_fee(fee):
     if type(fee) == str and fee.startswith("0x"):
         fee = int(fee, 0)
     assert get_latest_base_fee(web3) == int(fee)
+
+
+def test_get_effective_gas_price__eth():
```
Comment (mitche50) [inline tests/test_utils/test_tx_utils.py:118]: Do we want to add unhappy for eth and arb?  Looks like only set up for polygon currently

Comment (SHAKOTN) [reply tests/test_utils/test_tx_utils.py:118]: Yep, so unhappy path there was added just because polygon is using external API to calc gas

Comment (SHAKOTN) [reply tests/test_utils/test_tx_utils.py:118]: For other chains we don't call http client here

Comment (mitche50) [reply tests/test_utils/test_tx_utils.py:118]: Makes sense 👍

**Review by mitche50 — APPROVED** [review]

**Pull request merged**